www/docs/en/dev/guide/platforms/android/webview.md

```diff
@@ -29,106 +29,73 @@ components can communicate with each other, see Application Plugins.
+platform relies on a `SystemWebView` component.
 
 1. To follow these instructions, make sure you have the latest Cordova
    distribution. Download it from
    [cordova.apache.org](http://cordova.apache.org) and unzip its
    Android package.
 
-1. Navigate to the Android package's `/framework` directory and run
-   `ant jar`. It creates the Cordova `.jar` file, formed as
-   `/framework/cordova-x.x.x.jar`.
+1. Create a project in Android Studio
 
-1. Copy the `.jar` file into the Android project's `/libs` directory.
+1. Copy the framework directory into your project's root directory and name it
+CordovaLib
+
+1. Add that project as a library project to your main project.
 
 1. Add the following to the application's `/res/xml/main.xml` file,
    with the `layout_height`, `layout_width` and `id` modified to suit
    the application:
 
-        <org.apache.cordova.CordovaWebView
+        <org.apache.cordova.SystemWebView
             android:id="@+id/tutorialView"
             android:layout_width="match_parent"
             android:layout_height="match_parent" />
 
-1. Modify the activity so that it implements the `CordovaInterface`.
-   It should implement the included methods.  You may wish to copy
-   them from `/framework/src/org/apache/cordova/CordovaActivity.java`,
-   or else implement them on your own.  The following code fragment
-   shows a basic application that relies on the interface. Note how
-   the referenced view id matches the `id` attribute specified in the
-   XML fragment shown above:
-
-        public class CordovaViewTestActivity extends Activity implements CordovaInterface {
-            CordovaWebView cwv;
-            /* Called when the activity is first created. */
-            @Override
-            public void onCreate(Bundle savedInstanceState) {
-                super.onCreate(savedInstanceState);
-                setContentView(R.layout.main);
-                cwv = (CordovaWebView) findViewById(R.id.tutorialView);
-                Config.init(this);
-                cwv.loadUrl(Config.getStartUrl());
-            }
+1. Add the following code to setup your Activity so that you have an interface
 
-1. If the application needs to use the camera, implement the
-   following:
+        private CordovaInterfaceImpl iface = new CordovaInterfaceImpl(this);
+
+1. Add the following to intialize the WebView
+
+        //Set up the webview
+        ConfigXmlParser parser = new ConfigXmlParser();
+        parser.parse(this);
+
+        SystemWebView webView = (SystemWebView) findViewById(R.id.WebViewComponent);
+        webInterface = new CordovaWebViewImpl(new SystemWebViewEngine(webView));
+        webInterface.init(iface, parser.getPluginEntries(), parser.getPreferences());
+
+        loadUrl(parser.getLaunchUrl());
+
+
+1. Copy the application's HTML and JavaScript files to the Android
+   project's `app/src/main/assets/www` directory.
+
+1. Copy the `config.xml` file from `/framework/res/xml` to the
```

Comment:
There is no config.xml on /framework/res/xml. I've found two, one on test/res/xml and another on /bin/templates/project/res/xml, I suppose the good one is the one on bin, right?